doc/source/ray-overview/examples.rst

```diff
@@ -104,21 +104,21 @@ Ray Examples
         Aviary toolkit serving live traffic for LLMs
 
     .. grid-item-card:: :bdg-success:`Tutorial`
-        :class-item: gallery-item pytorch training
```

Comment:
Intuitively, the `train` tag on this is right, @matthewdeng is there a reason why we don't include it in the Train examples page?

Reply:
@woshiyyya do you remember the reason for not including this?

Reply:
Overall, I think it makes sense to add the training label here. This is not included in the Ray Train-Examples page, probably because we already have the "Getting Started with PyTorch" user guide and want users to go there.

Reply:
Ok - I'll leave it in.